doc/examples/widgets/lib/main.dart

```diff
@@ -39,7 +39,7 @@ Anchor parseAnchor(String name) {
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
 
-  final dashbook = Dashbook();
+  final Dashbook dashbook = Dashbook();
```

Comment:
We don't have any lint rule enforcing this, but we went on the past removing all types that weren't necessary, so here we should keep the former code.

Reply:
This was different in the `null-safety` branch. Not sure why. Reverting in both.